recipes/curl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "7.48.0" %}
+
+package:
+  name:    curl
+  version: {{ version }}
+
+source:
+  fn: curl-{{ version}}.tar.bz2
+  url: http://curl.haxx.se/download/curl-{{ version }}.tar.bz2
+  sha256: 864e7819210b586d42c674a1fdd577ce75a78b3dda64c63565abe5aefd72c753
+
+build:
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    # python is a build requirement on Windows to resolve features.
+    - python    # [win]
+    - openssl   # [unix]
+    - zlib
+  run:
+    - openssl   # [unix]
+    - zlib
+
+test:
+  commands:
+    # curl help commands on Windows have non-zero status codes.  Need other test.
+    - curl --help                  # [not osx]
+    - curl-config --features
+    - curl-config --protocols
+
+about:
+  home: http://curl.haxx.se/
+  license: MIT/X derivate (http://curl.haxx.se/docs/copyright.html)
+  license_family: MIT
+  license_file: COPYING
+  summary: tool and library for transferring data with URL syntax
+
+extra:
+  recipe-maintainers:
+    - msarahan
```

Comment:
@ocefpaf, given how important this is to you, would you mind being added as a maintainer here?